android/src/main/java/io/wazo/callkeep/VoiceConnection.java

```diff
@@ -53,12 +53,16 @@
         if (name != null && !name.equals("")) {
             setCallerDisplayName(name, TelecomManager.PRESENTATION_ALLOWED);
         }
+        Log.e(TAG, "Constructor");
```

Comment:
Why this log?

Reply:
Probably leftover debug log. Will remove.